runtime/databricks/automl_runtime/sklearn/one_hot_transformer.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2021 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from sklearn.base import TransformerMixin, BaseEstimator
+
+from scipy import sparse
+from category_encoders.one_hot import OneHotEncoder
+
+class OneHotTransformer(TransformerMixin, BaseEstimator):
+    def __init__(self, **kwargs):
+        self.sparse = True
+        if "sparse" in kwargs:
+            self.sparse = bool(kwargs["sparse"])
+            del kwargs["sparse"]
+        # TODO: we can make handle_unknown="indicator" as default here if we want to
```

Comment:
Can we file a story for this and link it in the TODO (e.g. TODO(ML-#####))?

Reply:
Actually, from what I've seen, I think I might have to update handle_unknown="indicator" here because handle_unknown=<something else> seems to lose information.

Reply:
But that might make it confusing for any future developments, so I will keep it as is here and have handle_unknown="indicator" be set in the universe code